eng/targets/Helix.props

```diff
@@ -20,7 +20,7 @@
     <IsMacHelixQueue Condition="$(HelixTargetQueue.Contains('OSX')) or $(HelixTargetQueue.Contains('macOs'))">true</IsMacHelixQueue>
     <HelixTestName>$(MSBuildProjectName)--$(TargetFramework)</HelixTestName>
     <LoggingTestingDisableFileLogging Condition="'$(IsHelixJob)' == 'true'">false</LoggingTestingDisableFileLogging>
-    <NodeVersion>10.15.3</NodeVersion>
+    <NodeVersion>14.17.6</NodeVersion>
```

Comment:
Let's send out a notification about this as it'll likely require updates to local node installs (https://nodejs.org/en/download/).

Reply:
Nobody should be running 10.15, I'll send an email to the team